src/tests/ftest/erasurecode/ec_offline_rebuild.yaml

```diff
@@ -22,7 +22,7 @@ hosts:
   test_clients:
     - client-A
     - client-B
-timeout: 900
+timeout: 2500
```

Comment:
That's a very long time out ....

Reply:
With partial IO some tests are taking 2384 seconds.

https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6625/3/artifact/Functional%20Hardware%20Large/erasurecode/ec_rebuild_disabled.py/results.html